recipe/bld.bat

```diff
@@ -0,0 +1,34 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+set "CFLAGS=%CFLAGS% -DWIN32 -DGSL_DLL"
+set "CXXFLAGS=%CXXFLAGS% -DWIN32 -DGSL_DLL"
```

Comment:
@czender GSL error was fixed with this

Reply:
Yep. The linking issue was fixed by @isuruf.

Also, I just added the extra test in my latest commit. See https://github.com/conda-forge/nco-feedstock/pull/48/commits/391c080f1b762cd97e3aa53beba495e20410decf